src/createServer.js

```diff
@@ -1,11 +1,23 @@
 'use strict';
 
-// const express = require('express');
+const { expenseController } = { ...require('./controllers/expenses') };
+const { userController } = { ...require('./controllers/users.js') };
+const { usersRouter } = { ...require('./routes/users.js') };
+const { expensesRouter } = { ...require('./routes/expenses.js') };
+const cors = require('cors');
+const express = require('express');
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  const app = express();
+
+  app.use(cors());
+  app.use(usersRouter);
```

Comment:
you can add here endpoint for user routes (you can delete `/users` from the user routes file)
```suggestion
  app.use('/users', usersRouter);
```